src/v0/destinations/iterable/data/IterableTrackConfig.json

```diff
@@ -28,7 +28,8 @@
   },
   {
     "destKey": "createdAt",
-    "sourceKeys": "originalTimestamp",
+    "sourceKeys": "createdAt",
```

Comment:
do we get createdAt all the time or should we have fallback to timestamp ?

Reply:
same here